core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnPredictionOp.scala

```diff
@@ -0,0 +1,39 @@
+package edu.uci.ics.texera.workflow.operators.sklearn
+
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+class SklearnPredictionOp extends PythonOperatorDescriptor {
```

Comment:
please keep the naming convention. `xxxOpDesc`.

Reply:
Fixed.